lib/floe/workflow/payload_template.rb

```diff
@@ -32,9 +32,7 @@ def parse_payload_array(value)
       def parse_payload_hash(value)
         value.to_h do |key, val|
           if key.end_with?(".$")
-            if value.key?(key.chomp(".$"))
-              raise Floe::InvalidWorkflowError, "both #{key} and #{key.chomp(".$")} present"
-            end
+            check_key_conflicts(value, key)
```

Comment:
super-extreme-minor, but my brain wants to see `key, value` in that order, and seeing it backwards is weird - can we flip these?